virt/logging_events_monitoring/virt-reviewing-virtualization-overview.adoc

```diff
@@ -0,0 +1,30 @@
+[id="virt-reviewing-virtualization-overview"]
+= Reviewing Virtualization Overview
```

Comment:
Suggestion: "Monitoring resources by using the Virtualization Overview page" or dashboard.
I think that might make the use case clearer. Otherwise, I am not sure the user would understand why it would be a good idea to review the Virt Overview pae.